src/internal/views/nunjucks/view-licences/tabs/bills.njk

```diff
@@ -0,0 +1,75 @@
+{% macro billLink(bill) %}
+<a href="#">
```

Comment:
Did you mean to leave the link out? I expect this link to take me to the invoice page which is part of the billing pages